roles/template_service_broker/defaults/main.yml

```diff
@@ -1,6 +1,24 @@
 ---
-# placeholder file?
 template_service_broker_remove: False
 template_service_broker_install: True
 openshift_template_service_broker_namespaces: ['openshift']
 template_service_broker_selector: "{{ openshift_hosted_infra_selector | default('region=infra') | map_from_pairs }}"
+
+__tsb_template_file: "apiserver-template.yaml"
```

Comment:
if you're going to move these to `defaults/main.yml` I vote dropping the preceeding `__`